index.js

```diff
@@ -54,29 +54,30 @@ const make = (files, options = {}) => {
 module.exports = (files, options) => {
 	const result = make(files, options);
 	const stdio = result.isTerminalEditor ? 'inherit' : 'ignore';
+	(async () => {
```

Comment:
Why are you wrapping it in async?

Reply:
You are right, there is no point to do so. I've pushed another commit to remove this.